### PR TITLE
Improve Docs

### PR DESCRIPTION
**Pull request by h0ngcha0**
Went through the docs and try to improve places that I feel should be improved.

Probably easier to review with local server (`npm start`)

### CODE REVIEW

File: docs/5min-overview.md [hunk 1]
```diff
@@ -9,14 +9,65 @@ import UntranslatedPageText from "@site/src/components/UntranslatedPageText";
...
+    title="Run Nodes"
+    icon={<RocketRegular />}
+    to="/full-node/getting-started"
+    description="Running Alephium full node is important for the security and decentralization of the network."
+  />
+  <Card
+    title="Wallets"
+    icon={<WalletRegular />}
+    to="/wallet/overview"
+    description="Running Alephium full node is important for the security and decentralization of the network."
```
Comment (Lbqds) [inline docs/5min-overview.md]: It should be the wallet description instead of the full node.

File: docs/dapps/build-dapp-from-scratch.md [hunk 2]
```diff
@@ -52,12 +54,13 @@ Contract TokenFaucet(
...
-`mut balance` is a mutable value that keeps track of how many tokens are left in this faucet.
+The first four fields are immutable values required to serve the [IFungibleToken interface](https://github.com/alephium/alephium-web3/blob/master/packages/web3/std/fungible_token_interface.ral).
+`balance` is a mutable value that keeps track of how many tokens are left in this faucet.
 
-You can see that our contract emits an `event` and defines an `error` code. Read the following for more info on [events](https://wiki.alephium.org/ralph/getting-started#events) and [error handling](https://wiki.alephium.org/ralph/getting-started#error-handling).
+You can see that our contract emits an `event` and defines an `error`
+code. For more information please read [events](https://wiki.alephium.org/ralph/getting-started#events) and [error handling](https://wiki.alephium.org/ralph/getting-started#error-handling).
 
-This is followed by 5 access methods for the different contract's arguments.
+This is followed by 5 access methods for each of the contract's arguments.
```
Comment (Lbqds) [inline docs/dapps/build-dapp-from-scratch.md]: [Nit]: `arguments` -> `fields`.

File: docs/full-node/full-node-more.md [hunk 3]
```diff
@@ -61,9 +77,9 @@ alephium.api.network-interface = "0.0.0.0"
 This error often occurs due to connection loss during node synchronization and means that some of the files are corrupted.
 To fix the problem:
 
-1. Delete the folder .alephium `rm .alephium`
+1. Delete the `db` folder under `${ALEPHIUM_HOME}/${NETWORK}`. e.g. `rm ~/.alephium/mainnet/db`
```
Comment (Lbqds) [inline docs/full-node/full-node-more.md]: [Nit]: `rm ~/.alephium/mainnet/db` -> `rm -rf ~/.alephium/mainnet/db`

File: docs/integration/integration-more.md [hunk 4]
```diff
@@ -165,3 +149,5 @@ Alephium implemented the same 51% attack protection as [Ravencoin](https://tronb
 ### What is the maximum reorg/rollback possible on Alephium?
 
 The maximum allowed reorg size is 100 blocks. It is recommend to check 110 blocks for higher safety.
+
```
Comment (Lbqds) [inline docs/integration/integration-more.md:152]: Perhaps we can mention that after the Rhone upgrade, the maximum reorg size is 200.

**Review by Lbqds — APPROVED** [review]
LGTM 👍

**Comment by polarker** [comment]
Amazing improvements 👍

**Pull request merged**